web/src/client/users.js

```diff
@@ -70,29 +70,30 @@ class UsersBaseClient {
   /**
    * Returns true if the root password is set
    *
-   * @return {Promise<Boolean>}
+   * @return {Promise<boolean>}
    */
   async isRootPasswordSet() {
     const proxy = await this.client.proxy(USERS_IFACE);
     return proxy.RootPasswordSet;
   }
 
   /**
-   * Sets the languages to install
+   * Sets the first user
    *
    * @param {User} user - object with full name, user name, password and boolean for autologin
-   * @return {Promise<boolean>} whether the operation was successful or not
+   * @return {Promise<object>} returns an object with the result and the issues found if error
```

Comment:
Please, annotate the type. I would expect this to be a common pattern (a result object returning the whether it was successful or not and a list of errors). So you could even create a `@type` for it.